CentralHub.Api.Model/Responses/Measurements/AggregatedMeasurements.cs

```diff
@@ -0,0 +1,71 @@
+using System.Text.Json.Serialization;
+
+namespace CentralHub.Api.Model.Responses.AggregatedMeasurements;
+
+public sealed class AggregatedMeasurements
+{
+    [JsonConstructor]
+    public AggregatedMeasurements(
+        int measurementDtoId,
+        DateTime startMeasurementTime,
+        DateTime endMeasurementTime,
+        int measurementCount,
```

Comment:
We dont have to specify measurement here. The class is called measurement, that should be good enough